### PR TITLE
Update shim to jupyter-js-services 0.5 API

### PR DESCRIPTION
**Pull request by jhpedemonte**
These changes are working for me. Not sure if more is needed.

Ref #381

### CODE REVIEW

**Comment by jdfreder** [comment]
@jhpedemonte your commit isn't credited to yourself.  Do you want me to merge anyways or would you like to fix that?

File: jupyter-js-widgets/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "npm-run-all": "^1.5.1",
     "rimraf": "^2.4.1",
     "sinon": "^1.17.2",
+    "sinon-chai": "^2.8.0",
```
Comment (jdfreder) [inline jupyter-js-widgets/package.json]: Why was this needed?

Comment (jhpedemonte) [inline jupyter-js-widgets/package.json]: I got an error running the tests, saying it couldn't find that module. After including it as dev-dependency, tests ran.

**Comment by jhpedemonte** [comment]
> your commit isn't credited to yourself.

Fixed. Thanks.

File: jupyter-js-widgets/package.json (hunk 1 above)
Comment (jdfreder) [inline jupyter-js-widgets/package.json]: Strange, I haven't seen that problem...

**Comment by jdfreder** [comment]
Thanks for doing this!
